book/doctrine.rst

```diff
@@ -499,16 +501,19 @@ of the bundle::
     public function createAction()
     {
         $product = new Product();
-        $product->setName('A Foo Bar');
-        $product->setPrice('19.99');
-        $product->setDescription('Lorem ipsum dolor');
+        $product->setName('Keyboard');
+        $product->setPrice(19.99);
+        $product->setDescription('Ergonomic and stylish!');
 
         $em = $this->getDoctrine()->getManager();
 
+        // register the Product entity with Doctrine's entity manager.
         $em->persist($product);
+
+        // synchronize all the registered entities with the database.
```

Comment:
I re-worded these (I like the commit in the comments) - it's just too technical imo for a beginner.